_episodes/l1-05-continuous_integration.md

```diff
@@ -24,32 +24,468 @@ keypoints:
+different repository entirely. See [GitHub Market Place: Checkout Action][checkout] for
+details.
+
+[checkout]: https://github.com/marketplace/actions/checkout
+
+The second step does not use a pre-packaged action but instead has a `run` entry. This
+allows us to execute some custom code. As a general rule, if you can find a pre-packaged
+action in the Marketplace that does what you want, use it, and only fall back to running
+custom code if necessary. For more detail on custom job steps see [GitHub Docs: Job Step
+Workflow Syntax][step-syntax].
+
+[step-syntax]: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun
+
+Once those two steps have completed, the CI run is finished. What happens next
+depends on what happened during the job steps. If any step did not finish successfully,
+but instead generated an error, then the CI run is considered to have failed. Successful
+CI runs are marked in the GitHub UI with a green dot next to the commit; failed runs
+have a red dot.
+
+> ## Adding CI to Your Recipe
+>
+> Let's look at adding some useful CI to the recipe repository. We're working with
+> Markdown files so it would be helpful to enforce a consistent style to avoid
+> differences between authors. Well do this by adding a workflow that runs the
+> `markdownlint-cli` action. This action runs [markdownlint-cli], a tool that checks
+> markdown files against a set of criteria.
+>
+> 1. Create a `.github` directory in your project then create a `workflows` directory
+>    within that.
+> 1. Create a file called `ci.yml` in the `workflows` directory.
+> 1. Add the following contents to `ci.yml`:
+>
+>     ```yaml
+>     on:
+>       - push
+>     jobs:
+>       markdownlint:
+>         runs-on: ubuntu-latest
+>         steps:
+>           - uses: actions/checkout@v3
+>           - name: markdownlint-cli
+>             uses: nosborn/github-action-markdown-cli@v3.2.0
+>             with:
+>               files: .
+>     ```
+>
+> 1. Stage and commit `ci.yml` then push the repository to GitHub.
+> 1. Your first CI run should have been triggered! Quickly, go to your repository on
+>    GitHub and select the `Actions` tab. You should see a workflow with a glowing amber
+>    dot next to the commit messaged you provided. This means that the workflow is
+>    running.
+> 1. Click on the commit message. You now get a breakdown of the individual jobs within
+>    your workflow. It's only one job in this case - `markdownlint` - click on it to see
+>    its progress. You can see the individual steps, and the output that they produce as
+>    they run.
+> 1. Before long the workflow will complete but, alas, it should be a failure. Go back
+>    to the front page of the repository by clicking the `Code` tab. You should see your
+>    commit marked with a red cross to indicate that it failed the CI. You should also
+>    receive a notification (after a few minutes) via the email address associated with
```

Comment:
Is this the default? I do not receive notifications because I disabled it long ago, but I mention it just incase someone else is in the same situation and is surprised at not receiving the email.

Reply:
It does seem to be the default. Not sure if it's worth a note here.

Reply:
Maybe a small callout, just in case.